x/gov/src/params.rs

```diff
@@ -195,9 +199,9 @@ impl From<DepositParams> for inner::DepositParams {
         Self {
             min_deposit: min_deposit.into_iter().map(|e| e.into()).collect(),
             max_deposit_period: Some(inner::Duration {
-                seconds: max_deposit_period.as_secs() as i64,
-                nanos: max_deposit_period.subsec_nanos() as i32,
-            }), // TODO:NOW
+                seconds: max_deposit_period.duration_seconds().into(),
+                nanos: max_deposit_period.nanoseconds().into(),
```

Comment:
`From` for structure?

Reply:
I don't understand your question.

Reply:
Instead of parsing fields would it be better to implement `From` for whole structure.  `From<Duration> for inner::Duration`

Reply:
OK, this would be good except that our `Duration` is defined inside Tendermint and we don't import `ibc_proto` types.